docs/backends/summary.rst

```diff
@@ -22,7 +22,7 @@ Class                            Response wrapper             Supported stream t
 ``AkkaHttpBackend``              ``scala.concurrent.Future``  ``akka.stream.scaladsl.Source[ByteString, Any]``
 ``AsyncHttpClientFutureBackend`` ``scala.concurrent.Future``  n/a
 ``AsyncHttpClientScalazBackend`` ``scalaz.concurrent.Task``   n/a
-``AsyncHttpClientZioBackend``    ``scalaz.zio.IO``            n/a
+``AsyncHttpClientZioBackend``    ``scalaz.zio.IO``            Stream[Throwable, ByteBuffer]
```

Comment:
I think I would consider creating a separate module with the streams dependency - just as fs2 isn't part of the cats module (though it's the de facto streaming library for cats). People might not want to pull in the additional dependency. So maybe a `AsyncHttpClientZioStreamsBackend`?